src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Proxy agent code and analytics data
+tags:
+  - Browser monitoring
+  - Proxy
+metaDescription: "The agent can be configured to fetch its code and send analytics through a proxy server instead of New Relic's endpoints."
+---
+
+<Callout variant="important">
+This feature is currently available for those using the copy/paste or NPM browser installation methods. There is currently no UI or NerdGraph configuration options available. We are continuing to work on improving access to these and other configuration options.
+</Callout>
+
+Setting a proxy URL for the browser agent can be valuable because it can help you to get around ad blockers and security VPNs that block client requests to third-party domains.
+
+Ad blockers and security VPNs often block third-party domains because they can be used to track users across the web. By proxying your requests through a first-party subdomain, you can bypass these blockers and ensure that your observability data is still being collected.
+
+## How proxies work [#how-proxies-work]
+
+From [version 1.240.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, you can set up a proxy by configuring the following browser agent properties:
+* `init.proxy.assets` determines where to fetch the browser agent.
+  * If you’re using the copy/paste installation method, you’ll need to manually edit your JavaScript snippet and set `assets` to your domain URL. For example, if you’re using browser agent v1.240.0 Pro+SPA, and you set `init.proxy.assets` to `assets.mydomain.com`, the browser agent will be loaded from `https://assets.mydomain.com/nr-spa.<hash>-1.240.0.min.js` instead of `https://js-agent.newrelic.com/nr-spa.<hash>-1.240.0.min.js`.
```

Comment:
```suggestion
  * If you’re using the copy/paste installation method, you’ll need to manually edit your JavaScript snippet and set `assets` to your domain URL. 
  * For example, if you’re using browser agent v1.240.0 Pro+SPA, and you set `init.proxy.assets` to `assets.mydomain.com`, the browser agent will be loaded from `https://assets.mydomain.com/nr-spa.<hash>-1.240.0.min.js` instead of `https://js-agent.newrelic.com/nr-spa.<hash>-1.240.0.min.js`.
```